### PR TITLE
fix business rule list content

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: Backoffice/BusinessRules/Strategies/ContentTypeStrategy.php [hunk 1]
```diff
@@ -2,22 +2,36 @@
...
 use OpenOrchestra\Backoffice\Security\ContributionActionInterface;
 use OpenOrchestra\ModelInterface\Repository\ContentRepositoryInterface;
+use OpenOrchestra\ModelInterface\Repository\SiteRepositoryInterface;
 
 /**
  * class ContentTypeStrategy
  */
 class ContentTypeStrategy extends AbstractBusinessRulesStrategy
 {
+    CONST READ_LIST = 'READ_LIST';
```
Comment (ngilain) [inline Backoffice/BusinessRules/Strategies/ContentTypeStrategy.php]: @alavieille : I'n not a big fan of having this const here, it should be better to have it in ContributionActionInterface or something similar to get it available for other kind of subjects

Comment (alavieille) [inline Backoffice/BusinessRules/Strategies/ContentTypeStrategy.php]: @ngilain the probleme is more generic, i create a ticket for it

Comment (ngilain) [inline Backoffice/BusinessRules/Strategies/ContentTypeStrategy.php]: ok

**Comment by ngilain** [comment]
:+1:

**Pull request merged**